lms/djangoapps/teams/models.py

```diff
@@ -15,6 +15,7 @@ class CourseTeam(models.Model):
     """This model represents team related info."""
 
     team_id = models.CharField(max_length=255, unique=True)
+    discussion_id = models.CharField(max_length=255, unique=True)
```

Comment:
Should an index be specified to discussion_id? We will need to determine the CourseTeam associated with a discussion_id.